lms/djangoapps/course_goals/models.py

```diff
@@ -34,3 +38,17 @@ def __unicode__(self):
 
     class Meta:
         unique_together = ("user", "course_key")
+
+
+@receiver(models.signals.post_save, sender=CourseEnrollment, dispatch_uid="create_course_goal_on_enroll")
```

Comment:
I think these should live in signals.py.  These signals are not even tied to this model, so this is an unexpected home for them.

Reply:
Moved